Code/Framework/AzToolsFramework/AzToolsFramework/AssetBrowser/Views/AssetBrowserTreeView.cpp

```diff
@@ -573,6 +578,110 @@ namespace AzToolsFramework
+                            AZ::OSString(fromPath.c_str()), AZ::OSString(toPath.c_str()), AssetChangeReportRequest::ChangeType::CheckMove);
+                        AssetChangeReportResponse response;
+
+                        if (SendRequest(request, response))
+                        {
+                            AZStd::string message;
+                            for (int i = 0; i < response.m_lines.size(); ++i)
+                            {
+                                message += response.m_lines[i] + "\n";
+                            }
+
+                            if (message.size())
+                            {
+                                QMessageBox msgBox(this);
+                                msgBox.setWindowTitle("Before Move Asset Information");
+                                msgBox.setIcon(QMessageBox::Warning);
+                                msgBox.setText("The asset you are moving may be referenced in other assets.");
+                                msgBox.setInformativeText("More information can be found by pressing \"Show Details...\".");
+                                QAbstractButton* moveButton = (QAbstractButton*)msgBox.addButton("Move", QMessageBox::YesRole);
+                                msgBox.setStandardButtons(QMessageBox::Cancel);
+                                msgBox.setDefaultButton(QMessageBox::Yes);
+                                msgBox.setDetailedText(message.c_str());
+                                QSpacerItem* horizontalSpacer = new QSpacerItem(600, 0, QSizePolicy::Minimum, QSizePolicy::Expanding);
+                                QGridLayout* layout = (QGridLayout*)msgBox.layout();
+                                layout->addItem(horizontalSpacer, layout->rowCount(), 0, 1, layout->columnCount());
+                                msgBox.exec();
+
+                                if (msgBox.clickedButton() == moveButton)
+                                {
+                                    AssetChangeReportRequest request2(
+                                        AZ::OSString(fromPath.c_str()),
+                                        AZ::OSString(toPath.c_str()),
+                                        AssetChangeReportRequest::ChangeType::Move);
+                                    AssetChangeReportResponse response2;
+                                    if (SendRequest(request2, response2))
+                                    {
+                                        AZStd::string message2;
+                                        for (int i = 0; i < response2.m_lines.size(); ++i)
+                                        {
+                                            message2 += response2.m_lines[i] + "\n";
+                                        }
+
+                                        if (message2.size())
+                                        {
+                                            QMessageBox msgBox2(this);
```

Comment:
Same as above for `response2`, `message2`, and `msgBox2`

Reply:
Replaced with moveResponse, moveMessage, moveLayout and moveMsgBox.